external-crates/move/crates/move-compiler/tests/move_check/parser/invalid_tyarg_locs.exp

```diff
@@ -32,6 +32,18 @@ error[E01016]: invalid name
    │
    = Type arguments should only appear on module members
 
+error[E04007]: incompatible types
+   ┌─ tests/move_check/parser/invalid_tyarg_locs.move:23:26
+   │    
+23 │         fun test00(): S<u64> {
+   │                       ------ Expected: '0x42::m::S<u64>'
+   │ ╭────────────────────────────^
+   │ │ ╭──────────────────────────'
+24 │ │ │         0x42<u64>::m::S { u: 0 }
+25 │ │ │     }
+   │ ╰─│─────^ Invalid return expression
+   │   ╰─────' Given: '()'
```

Comment:
Why is it unit? Do we fail to parse this and give an empty expression? Can we log an UnresolvedError here instead? So more or less `Seq([UnresolvedError])`

Reply:
Yes, we fail to parse a function body altogether in two cases (likely because `0x42<u64>::m` parses differently than `a<u64>::m`, but logging `UnresolvedError` indeed fixes the problem here.